main/src/tutorial/tutorial.flix

```diff
@@ -0,0 +1,233 @@
+//
+// Welcome to the introductory Flix Tutorial!
+//
+// In this tutorial we will develop a simple intra-procedural dataflow analysis
+// that performs constant propagation for local variables. The program language
+// we consider is very simple, it has three types of statements:
+//
+//      x = c        [assignment statement]
+//      x = y + z    [addition statement]
+//      x = y / z    [division statement]
+//
+//
+// We begin by defining a namespace for all our declarations. We could have
+// used the root namespace, but to avoid clashes with other examples we
+// create a new namespace.
+//
+namespace Tutorial.CP {
+
+    //
+    // We will represent the program-under-analysis using three input
+    // relations. One for each type of statement:
+    //
+    rel LitStm(r: Str, c: Int)         // r = c
```

Comment:
If we renamed it to "assignment statement," then we should also change the name of the relation. It's still "LitStm" right now.